src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -1019,17 +1019,6 @@ static Library()
                 BuiltinFunctionsCore.Or,
                 Or
             },
-            {
-                BuiltinFunctionsCore.ParseJSON,
```

Comment:
If the function def has moved out of Core as well, probably need to update BuiltinFunctionsCore